src/extensions/snippets/SnippetsView/SnippetsWidgetBox.tsx

```diff
@@ -0,0 +1,184 @@
+import { GridList, GridListTile } from '@material-ui/core';
```

Comment:
The SnippetsWidgetBox is a single row in the snippets view corresponding to one of the selected units.